web/src/engine/osk/src/keyboard-layout/oskLayer.ts

```diff
@@ -76,6 +81,21 @@ export default class OSKLayer {
     this.capsKey     = this.findKey('K_CAPS');
     this.numKey      = this.findKey('K_NUMLOCK');
     this.scrollKey   = this.findKey('K_SCROLL');
+
+    if(this.spaceBarKey) {
+      const spacebarLabel = this.spaceBarKey.label;
+      let tButton = this.spaceBarKey.btn;
+
+      if (typeof (tButton.className) == 'undefined' || tButton.className == '') {
+        tButton.className = 'kmw-spacebar';
+      } else if (tButton.className.indexOf('kmw-spacebar') == -1) {
+        tButton.className += ' kmw-spacebar';
+      }
+
+      if (spacebarLabel.className != 'kmw-spacebar-caption') {
+        spacebarLabel.className = 'kmw-spacebar-caption';
+      }
```

Comment:
```suggestion
      spacebarLabel.className = 'kmw-spacebar-caption';
```

No need to guard this ... I am sure it'll already be optimized.